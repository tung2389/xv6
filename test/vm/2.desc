Test to write after protect