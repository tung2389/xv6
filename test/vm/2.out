pid 3 test_2: trap 14 err 7 on cpu 0 eip 0x1054 addr 0x1000--kill proc
